src/DSNConfigurator.php

```diff
@@ -218,4 +218,25 @@ private function configureOptions(PHPMailer $mailer, $options)
             }
         }
     }
+
+    /**
+     * Parse URL.
+     *
+     * @param string $url URL
+     *
+     * @return array Result
+     */
+    private function parseUrl($url)
```

Comment:
Most internal functions like this are protected rather than private, so that if they cause any trouble, they can be overridden without requiring changes in the package.

Reply:
@Synchro, Ok, I've changed method's visibility to `protected`.